projects/storefrontlib/src/cms-components/cart/cart-coupon/applied-coupons/applied-coupons.component.html

```diff
@@ -0,0 +1,45 @@
+<div *ngIf="isReadOnly; else editableCoupons">
+  <div *ngIf="sortedVouchers.length > 0">
+    <div class="cx-applied-coupon-title" data-test="summary-title-coupon">
+      {{ 'voucher.coupon' | cxTranslate: { count: sortedVouchers.length } }}
+    </div>
+  </div>
+  <div
+    *ngFor="let v of sortedVouchers"

```

Comment:
`voucher` instead of `v` would be little more readable, what do you think?

Reply:
Fixed